config/keystorm.yml

```diff
@@ -1,14 +1,16 @@
-
 defaults: &defaults
   opennebula:
     endpoint: http://localhost:2633/RPC2
     secret: oneadmin:opennebula
     api-call-timeout: 3h
   log_level: debug
-  token_cipher: AES_128_CBC
-  token_key: abcdefghijklmnoprstuvxyz
-  token_iv: 123456789
-  default_email: nomail@nomail.com
+  token_cipher: AES-128-CBC
+  token_key: bbbbbbbbbbbbbbbb
+  token_iv: aaaaaaaaaaaaaaaa
+  oidc:
+    group_path: urn:mace:egi.eu:aai.egi.eu:{role}@{group}
```

Comment:
I don't know if `group_path` is the right name for this ... in what way do you see this as something `path`-like? Perhaps `pattern` or `matcher` or `mapper` or `filter`? However, this is just semantics. If you feel strongly about `path`, I can live with that :-)